src/pages/modules/LoginPhase/submodules/TokenInput/TokenInput.test.js

```diff
@@ -18,7 +18,7 @@ describe('TokenInput.js', () => {
   });
 
   afterEach(() => {
-    jest.runAllTimers();
+    jest.advanceTimersByTime(10000);
```

Comment:
setInterval was being accessed in this test suite (assumedly), so doing `runAllTimers` was causing infinite looping.